apps/cli/src/commands/exec.ts

```diff
@@ -7,32 +7,31 @@ import path from "path";
 export default class Exec extends Command {
     static description = "Execute command in cartesi machine of application";
 
-    static examples = ["<%= config.bin %> <%= command.id %>"];
+    static examples = ["<%= config.bin %> <%= command.id %> /bin/bash"];
 
     static args = {
         image: Args.string({
             description: "image ID|name",
             required: false,
         }),
+        command: Args.string({
```

Comment:
What is the image arg above

Reply:
I think we can set a custom image name with it